package.json

```diff
@@ -79,7 +79,7 @@
     "svgo": "^1.0.3",
     "sw-precache-webpack-plugin": "0.11.4",
     "url-loader": "0.6.2",
-    "vanilla-framework": "^1.6.2",
+    "vanilla-framework": "1.6.3",
```

Comment:
This should have been updating to 1.6.3 already but locking it to at least 1.6.3 is more reliable.
If we want to limit it a bit more, we could use `~1.6.2` to include point releases.
Otherwise I think we should keep the `^`